### ISSUE TITLE
NTR: Inositol polyphosphate 5-phosphatase OCRL-1 in fibroblasts low (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9071]
New term request Inositol polyphosphate 5-phosphatase OCRL-1 in fibroblasts low (segal_020930140412):

**Comment by pnrobinson** [comment]
@MickeySegal
does this have a disease context?

**Comment by MickeySegal** [comment]
https://www.ncbi.nlm.nih.gov/books/NBK1480/ "If a variant of unknown significance or no pathogenic variant is identified in a male with a clinical diagnosis consistent with Lowe syndrome, testing of inositol polyphosphate 5-phosphatase OCRL-1 activity in cultured skin fibroblasts is a possible option."

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000615
- label: Reduced Inositol polyphosphate 5-phosphatase OCRL-1 activity in cultured fibroblasts
- parents: [Abnormal enzyme activity in cultured fibroblasts (HP:4000195)](https://hpo.jax.org/app/browse/term/HP:4000195)
- pmid: PMID:20301653
- def: Concentration or activity of inositol polyphosphate 5-phosphatase OCRL-1 as measured in cultured fibroblasts is below the limits of normal. This enzyme has phosphoinositide 5-phosphatase (EC 3.1.3.36) activity.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9071
- synonyms:
